include/PassType.h

```diff
@@ -0,0 +1,61 @@
+#ifndef PASS_H
+#define PASS_H
+
+#include "RunAnalysis.h"
+#include "Utils.h"
+#include "CFG.h"
+
+#include <set>
+#include <fstream>
+
+// mapping between LLVM intermediate variables (e.g., %6, %7, %8) and variables visible in the code (e.g., %str). Used to find all local must-aliases, though it is not expected to find all of them
```

Comment:
"used to final **all** local must-aliases, though it is not expected to find **all** of them" (emphasis mine) is contradictory. I'd remove the first "all"